tests/unit/test_copies.py

```diff
@@ -133,6 +137,122 @@ def test_main(self):
         )
         self.stubber.assert_no_pending_responses()
 
+    def test_main_checksum_crc32(self):
```

Comment:
We may be able to consolidate these tests to be less specific for the type of checksum. A test that focuses on how we translate arbitrary checksums should be able to cover the change. You can find some example tests in the [upload change](https://github.com/boto/s3transfer/commit/36ea2e538bf24f29a5f546520b9b03d2db130c68) for reference.

Reply:
Consolidated the test into just one in 81a4c339f8d810b6737122b8999a9f4b00402e0c